app/modules/submissions/resources.py

```diff
@@ -244,10 +312,10 @@ def delete(self, submission):
         """
         Delete a Submission by ID.
         """
-        submission, submission_guids = submission
+        submission = self._get_submission_with_428(submission)
 
         if submission is None:
-            # We have checked the submission manager and cannot find this submission, raise 404 manually
+            # TODO Shouldn't this be a 204 No Content?
```

Comment:
@bluemellophone, I'm sure this should be a 204 but couldn't see how to raise that manually, suggestions welcome.

Reply:
Ignore this one, I've figured this one out